### PR TITLE
🐛Accept "ol" as a child of amp-sidebar > nav

### LINKED ISSUE #20312: Tag 'ol' is disallowed as child of tag 'amp-sidebar > nav'. Child tag must be one of ['ul'].
## What's the issue?
Consider markup:
```
<amp-sidebar id="sidebar1" layout="nodisplay" side="left">
<nav id="document-navigation"
             toolbar="(max-width: 641px)"
             toolbar-target="nav-target">
            <ol>
                <li>Item</li>
            </ol>
</nav>
</amp-sidebar>
```

```Tag 'ol' is disallowed as child of tag 'amp-sidebar > nav'. Child tag must be one of ['ul'].```

I find it strange that only `ul` should be supported. I reckon `ol` is just as valid.

## How do we reproduce the issue?

Use above source code.

## What browsers are affected?

All.

## Which AMP version is affected?
All.

### PR DESCRIPTION
**Pull request by caroqliu**
Resolves #20312 

- Validate `ol` as a child of `amp-sidebar > nav`.
- Add additional examples in validator tests

### CODE REVIEW

**Comment by honeybadgerdontcare** [comment]
@twifkak 	for validator review

File: extensions/amp-sidebar/validator-amp-sidebar.protoascii [hunk 1]
```diff
@@ -93,5 +93,6 @@ tags: {  # amp-sidebar > nav
   child_tags: {
     mandatory_num_child_tags: 1
     child_tag_name_oneof: "UL"
+    child_tag_name_oneof: "OL"
```
Comment (aghassemi) [inline extensions/amp-sidebar/validator-amp-sidebar.protoascii]: There is some text in `amp-sidebar.md` that also needs to be updated, namely: "The nav containing the toolbar attribute must only contain a single `<ul>`"

Comment (aghassemi) [inline extensions/amp-sidebar/validator-amp-sidebar.protoascii]: I think the CSS for `.i-amphtml-toolbar > ul` (in `amp-sidebar.css`) need to be updated as well.

maybe we can add two new `visual-tests` for `toolbar`? One for `ul` and one for `ol`?

Comment (twifkak) [inline extensions/amp-sidebar/validator-amp-sidebar.protoascii]: Prefer to alpha sort these lists. "OL" < "UL" :)

Comment (caroqliu) [reply extensions/amp-sidebar/validator-amp-sidebar.protoascii]: Sgtm, working on adding visual tests now.

File: extensions/amp-sidebar/amp-sidebar.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ You can create a `toolbar` element that displays in the `<body>` by specifying t
 - The sidebar may implement toolbars by adding nav elements with the `toolbar` attribute and `toolbar-target` attribute.
 - The nav element must be a child of `<amp-sidebar>` and follow this format: `<nav toolbar="(media-query)" toolbar-target="elementID">`.
     - For instance, this would be a valid use of toolbar: `<nav toolbar="(max-width: 1024px)" toolbar-target="target-element">`.
-- The nav containing the toolbar attribute must only contain a single `<ul>` element, that contains `<li>` elements.
+- The nav containing the toolbar attribute must only contain a single `<ul>` or `ol` element, that contains `<li>` elements.
```
Comment (cvializ) [inline extensions/amp-sidebar/amp-sidebar.md]: super nit: can you please put angle brackets around `ol` for consistency on this line with `<ul>` and `<li>`

**Pull request merged**